src/types.ts

```diff
@@ -285,3 +285,9 @@ export type NotifyStepResponse = {
    */
   notifyResponse: NotifyResponse[];
 };
+
+export type CallResponse = {
```

Comment:
I think this should have generics

Reply:
you mean for body field right? This is the same issue as whether we want to have `await context.call<SomeType>`. I would like to think a bit more before adding this